cwms-data-api/src/main/java/cwms/cda/api/ForecastTimeseriesController.java

```diff
@@ -0,0 +1,124 @@
+import static cwms.cda.api.Controllers.SIZE;
+import static cwms.cda.api.Controllers.SPEC_ID;
+import static cwms.cda.api.Controllers.TIMESERIES_ID;
+
+public class ForecastTimeseriesController implements CrudHandler {
+    private static final Logger logger = Logger.getLogger(ForecastSpecController.class.getName());
+
+    public static final String TAG = "Forecast";
+    private final MetricRegistry metrics;
+
+    private final Histogram requestResultSize;
+
+    public ForecastTimeseriesController(MetricRegistry metrics) {
+        this.metrics = metrics;
+        String className = this.getClass().getName();
+        requestResultSize = this.metrics.histogram((name(className, RESULTS, SIZE)));
+    }
+
+    private Timer.Context markAndTime(String subject) {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+
+    @OpenApi(
+            description = "Used to create and save a forecast timeseries",
+            requestBody = @OpenApiRequestBody(
+                    content = {
+                            @OpenApiContent(from = TimeSeries.class, type = Formats.JSONV2)
+                    },
+                    required = true
+            ),
+            queryParams = {
+                    @OpenApiParam(name = FORECAST_DATE_TIME, required = true, description = "Specifies the " +
+                            "forecast date time of the forecast instance to be associated with the created" +
+                            "forecast timeseries."),
+                    @OpenApiParam(name = ISSUE_DATE_TIME, required = true, description = "Specifies the " +
+                            "issue date time of the forecast instance to be associated with the created " +
+                            "forecast timeseries."),
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the " +
+                            "owning office of the forecast spec whose forecast instance will be " +
+                            "associated with the created forecast timeseries."),
+                    @OpenApiParam(name = SPEC_ID, required = true, description = "Specifies the " +
+                            "spec id of the forecast spec whose forecast instance will be " +
+                            "associated with the created forecast timeseries."),
+                    @OpenApiParam(name = LOCATION, required = true, description = "Specifies the " +
+                            "location of the forecast spec whose forecast instance will be" +
+                            "associated with the created forecast timeseries."),
+                    @OpenApiParam(name = TIMESERIES_ID, required = true, description = "Id of timeseries " +
+                            "that will be created.")
+            },
+            method = HttpMethod.POST,
+            path = "/forecast-timeseries",
+            tags = TAG
+    )
+    @Override
+    public void create(@NotNull Context ctx) {
```

Comment:
I'm re-thinking how this create would work.

Workflow:

client creates a spec, providing a list of time series ids
client creates an instance for a spec when going to run a forecast
client posts the results for the list of time series associated with the spec. -> isn't this just the time series controller? the spec is already associated with the time series identifier and the instance is already associated with the values given the forecast date and the time series id. What extra logic is needed by CDA to tie the time series storage to the instance?

Similarly for the delete, any interactions would be in the existing timeseries controller.